src/vs/workbench/contrib/debug/browser/breakpointWidget.ts

```diff
@@ -227,6 +236,29 @@ export class BreakpointWidget extends ZoneWidget implements IPrivateBreakpointWi
 				return suggestionsPromise;
 			}
 		}));
+
+		this.toDispose.push(this._configurationService.onDidChangeConfiguration((e) => {
+			if (e.affectsConfiguration('editor.fontSize')) {
```

Comment:
If `editor.lineHeight` changes we can also update

Reply:
Done.